### PR TITLE
Refactor code 

### PR DESCRIPTION
**Pull request by Dachrono**
# 🎉 🎈 In this *Milestone* 🙋‍♂️ 🚩 ‼ :

- Follow the best GitHub flow. ✅
- Document in a professional way. ✅
- Install linters. ✅
- No linters error. ✅

* Make sure YAGNI, KISS, and DRY rules and Ruby style guide are respected.
* Follow SOLID principles.
* Pay special attention to the "Single-responsibility Principle".
   - Make sure that the role of main.rb is only to start your code and ask the user for the initial choice of options (or to quit).
   - Extract some pieces of logic into separate class(es).

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -51,8 +60,7 @@ def create_person(role)
     puts "#{role.capitalize} '#{name}' created."
   end
 
-  # rubocop:enable Metrics/MethodLength
-  def create_book
+  def create_book_prompt
     puts 'Enter title:'
     title = gets.chomp
```
Comment (Gambit142) [inline app.rb]: - [ ] All codes responsible for receiving inputs or codes responsible for validating the input from the user should be put in a separate class or class method or module. The other methods for creating a class **_instance/objects_** or listing **_instances/objects_** should only create those instances. 👍 😎 

---

For Example, the `create_book_prompt` method should only create a new instance of the `Book` class while codes for getting the `title` and `author` of the _**book**_  from the console app should be put in a separate method, class or module. 💻 

--- 

Kindly apply the change suggested above for the following methods in the app.rb file:

- `create_person`
- `create_person_prompt`
- `create_book_prompt`
- `create_rental_prompt`
---

#### N.B: Please ensure that the app works as it should after refactoring the code. 👍

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @Dachrono @Kidd254 👋, 

Good job so far!

## Highlights
- All linters checks passed ✔️ 
- README file is professional ✔️

You still need to work on some issues to go to the next project, but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

N/A

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: app.rb (hunk 1 above)
Comment (Dachrono) [reply app.rb]: Solved

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @Kidd254, and @Dachrono👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- Good job with refactoring. ✔️
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- No linter errors. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**